pkgs/desktops/cinnamon/cinnamon-desktop/default.nix

```diff
@@ -0,0 +1,67 @@
+{ fetchFromGitHub
+, gdk-pixbuf
+, gobject-introspection
+, gtk3
+, intltool
+, meson
+, ninja
+, pkgconfig
+, pulseaudio
+, python3
+, stdenv
+, xkeyboard_config
+, xorg
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cinnamon-desktop";
+  version = "4.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "17hb8gkb9pfj56ckva5g4x83yvmdv7hvpidxjsdf79dw6pabr5rg";
+  };
+
+  buildInputs = [
```

Comment:
We're missing: 
* `xorg.libXext` https://github.com/linuxmint/cinnamon-desktop/blob/7113417186f2c4c8688f313935291daaf7cc2bcd/meson.build#L40
* `xorg.libXrandr` https://github.com/linuxmint/cinnamon-desktop/blob/7113417186f2c4c8688f313935291daaf7cc2bcd/meson.build#L43